### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by OneHitX** [BlackHoleMonster/IP-BlackHole#4] [Ideas]
A Discord server would be appreciated :)

**Comment by BlackHoleMonster** [comment]
Hello OneHitX :)

How the Discord server would be useful for us? 
Rather i setup IRCd.

**Reply by OneHitX** [reply]
Discord is a communication platform that can be useful for various purposes, including:

Connecting with others: Discord allows you to connect with people who share similar interests, whether it's gaming, music, art, or anything else. You can join or create servers that focus on specific topics, and engage in discussions with others who have similar interests.

Collaboration: Discord can be used for collaboration among teams or groups. You can create channels for specific projects or tasks, and communicate with team members in real-time. You can also use Discord's screen sharing feature to work on projects together, even if you're in different locations.

Learning: Discord can be a great tool for learning and education. You can join servers focused on specific topics, such as coding or language learning, and engage in discussions with other learners. You can also find and join study groups or tutoring services on Discord.

Community building: Discord can be used to build communities around your brand or content. You can create a server for your audience and engage with them directly, share updates, and host events or giveaways.

Overall, Discord can be a useful tool for communication, collaboration, learning, and community building. It offers a range of features that can be customized to meet your specific needs, making it a versatile platform for a variety of use cases.

**Reply by BlackHoleMonster** [reply]
Thanks, but it seems to me like there is no point at this time to have Discord server.
